app/components/dashboard/DcSunburstChart.vue

```
@@ -0,0 +1,598 @@
+<script lang="ts" setup>
+import * as dc from 'dc';
+import * as d3 from 'd3';
+import { onMounted, ref, watch, defineProps, defineExpose, defineEmits, defineModel, onUnmounted, computed } from "vue";
+import { useDcBaseChart } from '../../composables/dashboard/useDcBaseChart';
+
+// v-model for filters
+const filters = defineModel<Array<any>>('filters', {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)